centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/utils.ts

```diff
@@ -95,3 +95,19 @@ export const getResourcesUrlForMetricsWidgets = ({
 
   return `/monitoring/resources?details=${encodedDetailsParams}&filter=${encodedFilterParams}&fromTopCounter=true`;
 };
+
+const formatResource = (item): object => ({
+  ...item,
+  resources: map(pick(['id', 'name']), propOr([], 'resources', item))
+});
+
+export const formatLayoutResources = (data?: object): object | null => {
+  if (!data) {
+    return null;
+  }
+
+  return {
+    ...data,
+    resources: map(formatResource, propOr([], 'resources', data))
+  };
+};
```

Comment:
I think the best way to solve this issue is to update the Resources input directly (i.e update the changeResources and changeResource functions) by picking up only id/name properties.